src/test/java/org/jenkinsci/plugins/buildnamesetter/BuildNameSetterTest.java

```diff
@@ -78,6 +79,8 @@ public void shouldUse_default_config_values_if_null() throws InterruptedExceptio
 	private void asssertDisplayName(FreeStyleBuild build, String expectedName) {
 		assertEquals(Result.SUCCESS, build.getResult());
 		assertEquals(expectedName, build.getDisplayName());
+		EnvironmentVarSetter action = build.getAction(EnvironmentVarSetter.class);
+		assertEquals(expectedName, action.getVar(EnvironmentVarSetter.buildDisplayNameVar));

```

Comment:
Instead, you can test that environment variable was set. Presence of Action doesn't guarantee that environment variable will be set